recipes/leftbar/unread_chip/unread_chip.vue

```diff
@@ -67,9 +67,9 @@ export default {
 .dt-leftbar-unread-chip {
   &__mentions {
     .d-chip__label {
-      font-weight: var(--fw-medium);
-      background-color: var(--theme-mention-color-background);
-      color: var(--theme-mention-color-foreground);
+      font-weight: var(--fw-semibold);
+      background-color: var(--purple-400);
```

Comment:
Isn't this the same as --theme-mention-color-background?

Reply:
![Screenshot 2023-04-20 at 11 08 32 AM](https://user-images.githubusercontent.com/64808812/233451477-2b9320aa-3360-48cd-b173-1738b4eef363.png)

Reply:
same with the foreground.

Reply:
Yes, I was talking with @francisrupert yesterday and I'll refactor this component, I'll put this as Draft until I finish the changes